datadog-iam-user-handler/src/datadog_iam_user/handlers.py

```diff
@@ -0,0 +1,183 @@
+import logging
+from typing import Any, MutableMapping, Optional
+
+from cloudformation_cli_python_lib import (
+    Action,
+    HandlerErrorCode,
+    OperationStatus,
+    ProgressEvent,
+    Resource,
+    SessionProxy,
+)
+from datadog_api_client.v1 import ApiException
+from datadog_api_client.v1.api import users_api
+from datadog_api_client.v1.model.access_role import AccessRole
+from datadog_api_client.v1.model.user import User
+from datadog_cloudformation_common.api_clients import v1_client
+
+from .models import ResourceHandlerRequest, ResourceModel
+from .version import __version__
+
+# Use this logger to forward log messages to CloudWatch Logs.
+LOG = logging.getLogger(__name__)
+TYPE_NAME = "Datadog::IAM::User"
+
+resource = Resource(TYPE_NAME, ResourceModel)
+test_entrypoint = resource.test_entrypoint
+
+
+@resource.handler(Action.CREATE)
+def create_handler(
+    session: Optional[SessionProxy],
+    request: ResourceHandlerRequest,
+    callback_context: MutableMapping[str, Any],
+) -> ProgressEvent:
+    model = request.desiredResourceState
+    progress: ProgressEvent = ProgressEvent(
+        status=OperationStatus.IN_PROGRESS,
+        resourceModel=model,
+    )
+
+    LOG.info("Starting the User Resource Create Handler")
```

Comment:
nit: let's use the TYPE_NAME here 
```suggestion
    LOG.info("Starting %s Create Handler", TYPE_NAME)
```

Reply:
I went with f strings :)